[stable/pgadmin] upgrade to 4.17 and more

rowanruseler opened the pull request:
4.17:
  with 4.16 and up `USER: pgadmin` got added to the Dockerfile, this
  caused an ACL issue. securityContext and initContainers is added to
  resolve this issue.

other:
  - fixed dynamic PV provisioning
  - livenessProbe
  - readinessProbe
  - added more comments in values.yaml
  - updated README.md with new configuration items
  - chart bump version

Signed-off-by: Rowan Ruseler <rowanruseler@gmail.com>

rowanruseler committed:
[stable/pgadmin] upgrade to 4.17 and more

4.17:
  with 4.16 and up `USER: pgadmin` got added to the Dockerfile, this
  caused an ACL issue. securityContext and initContainers is added to
  resolve this issue.

other:
  - fixed dynamic PV provisioning
  - livenessProbe
  - readinessProbe
  - added more comments in values.yaml
  - updated README.md with new configuration items
  - chart bump version

Signed-off-by: Rowan Ruseler <rowanruseler@gmail.com>

davidkarlsen commented on stable/pgadmin/templates/deployment.yaml:
```diff
@@ -51,24 +61,32 @@ spec:
-        {{- toYaml . | nindent 8 }}
+        {{- .Values.affinity | toYaml | nindent 8 }}
     {{- end }}
-    {{- with .Values.tolerations }}
```
more of a personal taste - but I favour using `with` as it is easier to read and keep consistent due to no duplication.

rowanruseler replied:
Would you like me to adjust this? Personally I found this statement more clear. But I don't mind adjusting it back.

davidkarlsen commented on stable/pgadmin/templates/pvc.yaml:
```diff
@@ -1,19 +1,26 @@
-{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
```
See https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md for info on persistence

rowanruseler committed:
[stable/pgadmin] adjusted storageclass name

made a mistake with naming the variable different then what is shown in
the values.yml example

Signed-off-by: Rowan Ruseler <rowanruseler@gmail.com>

rowanruseler committed:
[stable/pgadmin]

resolved sessions bug by letting the probes go to the correct path

Signed-off-by: Rowan Ruseler <rowanruseler@gmail.com>

prydonius reviewed (approved):
/lgtm
